package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.91.0",
```

Comment:
I'm confused as to why this isn't causing a conflict. I bumped the version to 2.92 in a previous PR

Reply:
Hmm I guess since it is getting set to the same value, github is like - "nice, this is perfect! even less to do"